impl_days_room_info.html

```diff
@@ -0,0 +1,11 @@
+<div>
+    Currently we have a real time page listing all the ongoing projects here:
```

Comment:
Let's not commit the old links here. We will forget to properly change them. Let's just put a big todo in here.